examples/Django/emptyapi/config.yaml

```diff
@@ -0,0 +1,98 @@
+api_version: "0.1"
```

Comment:
Please, also make short form of this config. And if you skipped something here, then add field. I want to have full version and minimal version for each example.